tests/pactHelper.js

```diff
@@ -164,13 +194,7 @@ function setGeneralInteractions (provider) {
       body: Pact.Matchers.term({
         matcher: '<\\?xml version="1\\.0"\\?>\\s' +
           '<ocs>\\s' +
-          ' <meta>\\s' +
-          '  <status>ok<\\/status>\\s' +
-          '  <statuscode>100<\\/statuscode>\\s' +
-          '  <message>OK<\\/message>\\s' +
-          '  <totalitems><\\/totalitems>\\s' +
-          '  <itemsperpage><\\/itemsperpage>\\s' +
-          ' <\\/meta>\\s' +
+          ocsMeta('ok', 100, 'OK') +
```

Comment:
you've to also add pagination tags in `ocsMeata` or better use separate constant for paginated `xml` response
```js
'  <totalitems><\\/totalitems>\\s' +
'  <itemsperpage><\\/itemsperpage>\\s' +
```

Reply:
It is not required in most of the responses. If it is required later in any, we can just add it there in the specific response or can create a new var for that then